src/main/java/com/sigma/cinematicketpro/service/GenreService.java

```diff
@@ -0,0 +1,9 @@
+package com.sigma.cinematicketpro.service;
+
+import com.sigma.cinematicketpro.dto.tmdb.TMDBGenre;
+
+import java.util.Map;
+
+public interface GenreService {
+   Map<Integer, TMDBGenre> getAllGenresMap();
```

Comment:
this is a hak if you store them as id - id value genre then no need for custom repository method - just get all and map in service layer

Reply:
discussed